lib/eslint/eslint.js

```diff
@@ -746,7 +746,7 @@ class ESLint {
         });
         const controller = new AbortController();
         const retryCodes = new Set(["ENFILE", "EMFILE"]);
-        const retrier = new Retrier(error => retryCodes.has(error.code));
+        const retrier = new Retrier(error => retryCodes.has(error.code), { signal: controller.signal, concurrency: 100 });
```

Comment:
It doesn't seem like the constructor supports option `signal`?

https://github.com/humanwhocodes/retry/blob/90317d3f133d45583296ef740fb8f8e2997c3903/src/retrier.js#L244

Reply:
Doh! Put in the wrong place. Thanks.